examples/twittermap/web/public/javascripts/timeseries/controllers.js

```diff
@@ -120,11 +125,13 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
             $scope.queried = true;
             var ndx = $scope.ndx;
             if (ndx) {
-              ndx.remove();
-              ndx.add($scope.empty);
-              dc.redrawAll();
-              ndx.add(newVal);
-              dc.redrawAll();
+              if (!onPlay) {

```

Comment:
this if statement can be combined with `if(ndx)` using && operator, generally try to avoid nested if statement.

Reply:
They can't be combined because no matter what the value of onPlay is, it must return.